src/components/partner/dto/create-partner.dto.ts

```diff
@@ -1,8 +1,9 @@
-import { Field, InputType, ObjectType } from '@nestjs/graphql';
+import { Field, ID, InputType, ObjectType } from '@nestjs/graphql';
```

Comment:
Carson corrected me here...let's alias this instead of the other `ID`.

```suggestion
import { Field, ID as IDType, InputType, ObjectType } from '@nestjs/graphql';
```